packages/sdk/src/constants/CicsCmci.constants.ts

```diff
@@ -100,8 +100,27 @@ export const CicsCmciConstants = {
   PARAMETER: "PARAMETER",
 
   /**
-     * The CICS CMCI external resource names
+     * The CICS CMCI transaction definition
      */
-  CICS_CMCI_EXTERNAL_RESOURCES: ["CICSLocalTransaction", "CICSRemoteTransaction", "CICSDefinitionTransaction", "CICSLocalFile"],
+  CICS_CMCI_TRANSACTION_DEFINITION: "CICSDefinitionTransaction",
```

Comment:
For backward compatibility reasons, could we keep the `CICS_CMCI_EXTERNAL_RESOURCES`.
And add a `@deprecated` line in the JSDoc 🙏 

```
/**
     * The CICS CMCI external resource names
     * @deprecated
     */
     CICS_CMCI_EXTERNAL_RESOURCES: ["CICSLocalTransaction", "CICSRemoteTransaction", "CICSDefinitionTransaction", "CICSLocalFile"],
```